WordPress/WordPressTest/DataMigratorTests.swift

```diff
@@ -18,11 +19,14 @@ class DataMigratorTests: XCTestCase {
         keychainUtils = KeychainUtilsMock()
         sharedUserDefaults = InMemoryUserDefaults()
         localUserDefaults = InMemoryUserDefaults()
-        migrator = DataMigrator(coreDataStack: coreDataStack,
-                                backupLocation: URL(string: "/dev/null"),
-                                keychainUtils: keychainUtils,
-                                localDefaults: localUserDefaults,
-                                sharedDefaults: sharedUserDefaults)
+        migrator = DataMigrator(
+            coreDataStack: coreDataStack,
+            backupLocation: URL(string: "/dev/null"),
+            keychainUtils: keychainUtils,
+            localDefaults: localUserDefaults,
+            sharedDefaults: sharedUserDefaults,
+            appGroupName: appGroupName
+        )
```

Comment:
Look how better the code is this way!

The fact that the object uses the app group name in one way or another is clear in the `init`. Love it.